src/texture/mod.rs

```diff
@@ -358,6 +358,35 @@ pub struct RawImage3d<'a, T: Clone + 'a> {
     pub format: ClientFormat,
 }
 
+impl<'a, T: Clone + 'a> RawImage3d<'a, T> {
+    ///Transforms a Vec<RawImage2d> into a RawImage3d
+    pub fn from_vec_raw2d(arr: &Vec<RawImage2d<'a, T>>) -> RawImage3d<'a, T> {
+        let depth   = arr.len();
+        let width   = arr[0].width;
+        let height  = arr[0].height;
+        let format  = arr[0].format;
+        let clo = |v, w, h, d, f| {
```

Comment:
I think that has to do with me spending too much time on this, and not noticing that after the fact.